src/AerialShip/SamlSPBundle/Model/SSOState.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace AerialShip\SamlSPBundle\Model;
+
+/**
+ * @api
+ */
+class SSOState extends \AerialShip\SamlSPBundle\State\SSO\SSOState
+{
```

Comment:
I think the code on `\AerialShip\SamlSPBundle\State\SSO\SSOState` should be in here.
And keep `\AerialShip\SamlSPBundle\State\SSO\SSOState` extending `Model/SSOState` just for BC.